mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/MqttWebSocketPublisher.java

```diff
@@ -25,7 +27,7 @@ class MqttWebSocketPublisher implements MqttPublisher {
     MqttWebSocketPublisher(String webSocketBaseUrl, TokenApi tokenApi, MqttWebSocketConfig config) {
         this.webSocketBaseUrl = webSocketBaseUrl;
         this.config = config;
-        this.tokenSupplier = new TokenSupplier(tokenApi, config.getTopic(), SUBSCRIBER);
+        this.tokenSupplier = new TokenSupplier(tokenApi, config.getTopic(), getSubscriber());
```

Comment:
That's yet another thing which should be improved in our Notification 2.0 API - subscriber name is not really used in case of producer so maybe instead of random name to avoid any confusion just hardcode `notUsed` or something like that?

Reply:
Or separate methods/contracts for consumer Token method/object and producer one, the latter not having a name parameter

Reply:
subscriber field is required for TokenAPI. We can leave it blank though which would result in a random subscriber name anyway. Therefore, I would go just go with "mqttConnectPublisher".